options/locale/locale_en-US.ini

```diff
@@ -3530,7 +3530,7 @@ runs.actors_no_select = All actors
 runs.status_no_select = All status
 runs.no_results = No results matched.
 runs.no_runs = The workflow has no runs yet.
-runs.no_commit_message = empty commit message
+runs.empty_commit_message = empty commit message
```

Comment:
That's not what I meant. Now you have `actions.runs.empty_commit_message`, but I wanted `empty_commit_message` because the next time we need this translation outside of actions context, it will be a ugly usage.